src/python/grpcio_tests/tests/protoc_plugin/_split_definitions_test.py

```diff
@@ -165,11 +165,33 @@ def protoc(self, proto_path, python_out, absolute_proto_file_names):
         return pb2_grpc_protoc_exit_code, pb2_protoc_exit_code
 
 
+class _RelativeImportGrpcStyle(object):
+
+    def name(self):
+        return 'RelativeImportGrpcStyle'
+
+    def grpc_in_pb2_expected(self):
+        return False
+
+    def protoc(self, proto_path, python_out, absolute_proto_file_names):
```

Comment:
@gnossen I don't know if this is better or worse. Since most of the protoc related flavorings happen in the Style objects, I made a new one that will strip the `split_messages.sub` package. I froze the test run and verified the files show the correct output - this should raise some flavor of import error if the import statement becomes invalid again!

Reply:
(Another related bonus, is it runs all existing tests against this configuration, which is cool!)